### ISSUE TITLE
Cleanup Game Setup and Player Intialization

### ISSUE THREAD

**Issue by sIKE23** [sIKE23/Mage-Wars#141]
Looking at the code as it exists today, I think we should take a look at cleaning/refining/reducing how all of this is handled, Ideas, suggestions? 

The Roll Dice button needs to be created only once......

**Comment by anandaguneratne** [comment]
Well, first of all, I think reducing the number of choices would be a good idea. Better to make assumptions and then allow players to change things later for options that don't really affect the game (e.g. dice roll area, player color, etc.). If you don't want both players to be able to change things, we could have only the player who started the game be able to decide (if OCTGN has a way to give us that information).

**Comment by sIKE23** [comment]
I do like and think that players should be able to choose their color just like you can in real life. As for other items I think that we can pick defaults. Initiative is done as per the rules, i.e. winner chooses who goes first.

OCTGN really doesn't have the concept of the game owner or the player that started the game :(. 

I was just thinking that we "setup" the table after the player color is chosen. With the multiple card size coming available, the card anchoring (work is already done and you can DL the test ver on OCTGN to check it out), and the buttons that I want to add that the time is right to cleanup/streamline this aspect of the code.

**Comment by anandaguneratne** [comment]
Okay, here's a thought: game settings persist between games. So what if we allow players to select their preferences, store them, and then automatically apply them in each subsequent game (until they change them).

So for example, a player might store as a setting that he wants the dice roll area to be to the side. Then, in every game after that (unless he changes it), if he is determined to be the player who chooses where to place the dice roll area, it automatically assumes that he chooses the stored setting. This could remove several steps from setup.

**Comment by sIKE23** [comment]
Ok, when would we ask this question? I like this idea....

**Comment by sIKE23** [comment]
How about I create another Initiative Token color that we use during game setup and then change once we get the color selection done. This way we can create all of the buttons and game icons in one pass during OnTableLoad or OnGameStart.

**Comment by sIKE23** [comment]
Think I will leave this open and pull it forward into the Q2 Milestone.

**Comment by sIKE23** [comment]
I ended up pulling all of this work into Domination, and it is done!
